<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>655</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>DuckStation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/duckstation-qt.qrc">
    <normaloff>:/icons/duck.png</normaloff>:/icons/duck.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QStackedWidget" name="mainContainer"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>&amp;System</string>
    </property>
    <widget class="QMenu" name="menuChangeDisc">
     <property name="title">
      <string>Change Disc</string>
     </property>
     <property name="icon">
      <iconset theme="disc-line"/>
     </property>
     <actiongroup name="actionGroupChangeDiscSubImages"/>
     <addaction name="actionChangeDiscFromFile"/>
     <addaction name="actionChangeDiscFromDevice"/>
     <addaction name="actionChangeDiscFromGameList"/>
     <addaction name="actionRemoveDisc"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuCheats">
     <property name="title">
      <string>Cheats</string>
     </property>
     <property name="icon">
      <iconset theme="cheats-line"/>
     </property>
    </widget>
    <widget class="QMenu" name="menuLoadState">
     <property name="title">
      <string>Load State</string>
     </property>
     <property name="icon">
      <iconset theme="folder-open-line"/>
     </property>
    </widget>
    <widget class="QMenu" name="menuSaveState">
     <property name="title">
      <string>Save State</string>
     </property>
     <property name="icon">
      <iconset theme="save-3-line"/>
     </property>
    </widget>
    <addaction name="actionStartFile"/>
    <addaction name="actionStartDisc"/>
    <addaction name="actionStartBios"/>
    <addaction name="actionStartFullscreenUI"/>
    <addaction name="actionResumeLastState"/>
    <addaction name="separator"/>
    <addaction name="actionPowerOff"/>
    <addaction name="actionPowerOffWithoutSaving"/>
    <addaction name="actionReset"/>
    <addaction name="actionPause"/>
    <addaction name="menuChangeDisc"/>
    <addaction name="separator"/>
    <addaction name="menuCheats"/>
    <addaction name="actionScreenshot"/>
    <addaction name="separator"/>
    <addaction name="menuLoadState"/>
    <addaction name="menuSaveState"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>S&amp;ettings</string>
    </property>
    <addaction name="actionViewGameProperties"/>
    <addaction name="separator"/>
    <addaction name="actionInterfaceSettings"/>
    <addaction name="actionGameListSettings"/>
    <addaction name="actionBIOSSettings"/>
    <addaction name="actionConsoleSettings"/>
    <addaction name="actionEmulationSettings"/>
    <addaction name="actionMemoryCardSettings"/>
    <addaction name="actionGraphicsSettings"/>
    <addaction name="actionPostProcessingSettings"/>
    <addaction name="actionAudioSettings"/>
    <addaction name="actionAchievementSettings"/>
    <addaction name="actionFolderSettings"/>
    <addaction name="actionAdvancedSettings"/>
    <addaction name="separator"/>
    <addaction name="actionControllerSettings"/>
    <addaction name="actionHotkeySettings"/>
    <addaction name="separator"/>
    <addaction name="actionAddGameDirectory"/>
    <addaction name="actionScanForNewGames"/>
    <addaction name="actionRescanAllGames"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionGitHubRepository"/>
    <addaction name="actionDiscordServer"/>
    <addaction name="separator"/>
    <addaction name="actionCheckForUpdates"/>
    <addaction name="separator"/>
    <addaction name="actionViewThirdPartyNotices"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <widget class="QMenu" name="menuRenderer">
     <property name="title">
      <string>Switch GPU Renderer</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuCPUExecutionMode">
     <property name="title">
      <string>Switch CPU Emulation Mode</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuCropMode">
     <property name="title">
      <string>Switch Crop Mode</string>
     </property>
    </widget>
    <addaction name="menuCPUExecutionMode"/>
    <addaction name="menuRenderer"/>
    <addaction name="menuCropMode"/>
    <addaction name="actionDisableAllEnhancements"/>
    <addaction name="separator"/>
    <addaction name="actionCPUDebugger"/>
    <addaction name="separator"/>
    <addaction name="actionDumpRAM"/>
    <addaction name="actionDumpVRAM"/>
    <addaction name="actionDumpSPURAM"/>
    <addaction name="separator"/>
    <addaction name="actionDebugDumpCPUtoVRAMCopies"/>
    <addaction name="actionDebugDumpVRAMtoCPUCopies"/>
    <addaction name="separator"/>
    <addaction name="actionDebugShowVRAM"/>
    <addaction name="actionDebugShowGPUState"/>
    <addaction name="actionDebugShowCDROMState"/>
    <addaction name="actionDebugShowSPUState"/>
    <addaction name="actionDebugShowTimersState"/>
    <addaction name="actionDebugShowMDECState"/>
    <addaction name="actionDebugShowDMAState"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuWindowSize">
     <property name="title">
      <string>&amp;Window Size</string>
     </property>
    </widget>
    <addaction name="actionViewToolbar"/>
    <addaction name="actionViewLockToolbar"/>
    <addaction name="actionViewStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionViewGameList"/>
    <addaction name="actionViewGameGrid"/>
    <addaction name="actionViewSystemDisplay"/>
    <addaction name="separator"/>
    <addaction name="actionFullscreen"/>
    <addaction name="menuWindowSize"/>
    <addaction name="separator"/>
    <addaction name="actionMergeDiscSets"/>
    <addaction name="actionShowGameIcons"/>
    <addaction name="actionGridViewShowTitles"/>
    <addaction name="actionGridViewZoomIn"/>
    <addaction name="actionGridViewZoomOut"/>
    <addaction name="actionGridViewRefreshCovers"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionOpenDataDirectory"/>
    <addaction name="separator"/>
    <addaction name="actionMemoryCardEditor"/>
    <addaction name="actionCoverDownloader"/>
    <addaction name="separator"/>
    <addaction name="actionMemoryScanner"/>
    <addaction name="separator"/>
    <addaction name="actionMediaCapture"/>
    <addaction name="separator"/>
    <addaction name="actionOpenTextureDirectory"/>
    <addaction name="actionReloadTextureReplacements"/>
   </widget>
   <addaction name="menuSystem"/>
   <addaction name="menuSettings"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStartFile"/>
   <addaction name="actionStartBios"/>
   <addaction name="actionStartFullscreenUI2"/>
   <addaction name="separator"/>
   <addaction name="actionResumeLastState"/>
   <addaction name="actionReset"/>
   <addaction name="actionPause"/>
   <addaction name="actionChangeDisc"/>
   <addaction name="actionCheatsToolbar"/>
   <addaction name="actionScreenshot"/>
   <addaction name="separator"/>
   <addaction name="actionLoadState"/>
   <addaction name="actionSaveState"/>
   <addaction name="separator"/>
   <addaction name="actionFullscreen"/>
   <addaction name="actionSettings2"/>
   <addaction name="actionControllerSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStartFile">
   <property name="icon">
    <iconset theme="file-line"/>
   </property>
   <property name="text">
    <string>Start &amp;File...</string>
   </property>
  </action>
  <action name="actionStartDisc">
   <property name="icon">
    <iconset theme="disc-line"/>
   </property>
   <property name="text">
    <string>Start &amp;Disc...</string>
   </property>
  </action>
  <action name="actionStartBios">
   <property name="icon">
    <iconset theme="chip-line"/>
   </property>
   <property name="text">
    <string>Start &amp;BIOS</string>
   </property>
  </action>
  <action name="actionScanForNewGames">
   <property name="icon">
    <iconset theme="file-search-line"/>
   </property>
   <property name="text">
    <string>&amp;Scan For New Games</string>
   </property>
  </action>
  <action name="actionRescanAllGames">
   <property name="icon">
    <iconset theme="refresh-line"/>
   </property>
   <property name="text">
    <string>&amp;Rescan All Games</string>
   </property>
  </action>
  <action name="actionPowerOff">
   <property name="icon">
    <iconset theme="shut-down-line"/>
   </property>
   <property name="text">
    <string>Power &amp;Off</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset theme="restart-line"/>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="pause-line"/>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="actionLoadState">
   <property name="icon">
    <iconset theme="folder-open-line"/>
   </property>
   <property name="text">
    <string>&amp;Load State</string>
   </property>
  </action>
  <action name="actionSaveState">
   <property name="icon">
    <iconset theme="save-3-line"/>
   </property>
   <property name="text">
    <string>&amp;Save State</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="door-open-line"/>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionBIOSSettings">
   <property name="icon">
    <iconset theme="chip-line"/>
   </property>
   <property name="text">
    <string>B&amp;IOS</string>
   </property>
  </action>
  <action name="actionConsoleSettings">
   <property name="icon">
    <iconset theme="chip-2-line"/>
   </property>
   <property name="text">
    <string>C&amp;onsole</string>
   </property>
  </action>
  <action name="actionEmulationSettings">
   <property name="icon">
    <iconset theme="emulation-line"/>
   </property>
   <property name="text">
    <string>E&amp;mulation</string>
   </property>
  </action>
  <action name="actionControllerSettings">
   <property name="icon">
    <iconset theme="controller-digital-line"/>
   </property>
   <property name="text">
    <string>&amp;Controllers</string>
   </property>
  </action>
  <action name="actionHotkeySettings">
   <property name="icon">
    <iconset theme="keyboard-line"/>
   </property>
   <property name="text">
    <string>&amp;Hotkeys</string>
   </property>
  </action>
  <action name="actionGraphicsSettings">
   <property name="icon">
    <iconset theme="image-fill"/>
   </property>
   <property name="text">
    <string>&amp;Graphics</string>
   </property>
  </action>
  <action name="actionPostProcessingSettings">
   <property name="icon">
    <iconset theme="sun-fill"/>
   </property>
   <property name="text">
    <string>&amp;Post-Processing</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="icon">
    <iconset theme="fullscreen-line"/>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionResolution_Scale">
   <property name="text">
    <string>Resolution Scale</string>
   </property>
  </action>
  <action name="actionGitHubRepository">
   <property name="icon">
    <iconset resource="resources/duckstation-qt.qrc">
     <normaloff>:/icons/github.png</normaloff>:/icons/github.png</iconset>
   </property>
   <property name="text">
    <string>&amp;GitHub Repository...</string>
   </property>
  </action>
  <action name="actionDiscordServer">
   <property name="icon">
    <iconset resource="resources/duckstation-qt.qrc">
     <normaloff>:/icons/discord.png</normaloff>:/icons/discord.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Discord Server...</string>
   </property>
  </action>
  <action name="actionCheckForUpdates">
   <property name="icon">
    <iconset theme="download-2-line"/>
   </property>
   <property name="text">
    <string>Check for &amp;Updates...</string>
   </property>
  </action>
  <action name="actionViewThirdPartyNotices">
   <property name="text">
    <string>View Third-Party Notices...</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="resources/duckstation-qt.qrc">
     <normaloff>:/icons/QT.png</normaloff>:/icons/QT.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources/duckstation-qt.qrc">
     <normaloff>:/icons/duck_64.png</normaloff>:/icons/duck_64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About DuckStation...</string>
   </property>
  </action>
  <action name="actionChangeDisc">
   <property name="icon">
    <iconset theme="disc-line"/>
   </property>
   <property name="text">
    <string>Change Disc...</string>
   </property>
  </action>
  <action name="actionCheatsToolbar">
   <property name="icon">
    <iconset theme="cheats-line"/>
   </property>
   <property name="text">
    <string>Cheats</string>
   </property>
  </action>
  <action name="actionAudioSettings">
   <property name="icon">
    <iconset theme="volume-up-line"/>
   </property>
   <property name="text">
    <string>Audio</string>
   </property>
  </action>
  <action name="actionAchievementSettings">
   <property name="icon">
    <iconset theme="trophy-line"/>
   </property>
   <property name="text">
    <string>Achievements</string>
   </property>
  </action>
  <action name="actionFolderSettings">
   <property name="icon">
    <iconset theme="folder-open-line"/>
   </property>
   <property name="text">
    <string>Folders</string>
   </property>
  </action>
  <action name="actionGameListSettings">
   <property name="icon">
    <iconset theme="folder-open-line"/>
   </property>
   <property name="text">
    <string>Game List</string>
   </property>
  </action>
  <action name="actionInterfaceSettings">
   <property name="icon">
    <iconset theme="settings-3-line"/>
   </property>
   <property name="text">
    <string>&amp;Interface</string>
   </property>
  </action>
  <action name="actionAdvancedSettings">
   <property name="icon">
    <iconset theme="alert-line"/>
   </property>
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="actionAddGameDirectory">
   <property name="icon">
    <iconset theme="folder-add-line"/>
   </property>
   <property name="text">
    <string>Add Game Directory...</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="settings-3-line"/>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionSettings2">
   <property name="icon">
    <iconset theme="settings-3-line"/>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionChangeDiscFromFile">
   <property name="text">
    <string>From File...</string>
   </property>
  </action>
  <action name="actionChangeDiscFromDevice">
   <property name="text">
    <string>From Device...</string>
   </property>
  </action>
  <action name="actionChangeDiscFromGameList">
   <property name="text">
    <string>From Game List...</string>
   </property>
  </action>
  <action name="actionRemoveDisc">
   <property name="text">
    <string>Remove Disc</string>
   </property>
  </action>
  <action name="actionResume_State">
   <property name="text">
    <string>Resume State</string>
   </property>
  </action>
  <action name="actionGlobal_State">
   <property name="text">
    <string>Global State</string>
   </property>
  </action>
  <action name="actionDebugShowVRAM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show VRAM</string>
   </property>
  </action>
  <action name="actionDebugDumpCPUtoVRAMCopies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dump CPU to VRAM Copies</string>
   </property>
  </action>
  <action name="actionDebugDumpVRAMtoCPUCopies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dump VRAM to CPU Copies</string>
   </property>
  </action>
  <action name="actionDisableAllEnhancements">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disable All Enhancements</string>
   </property>
  </action>
  <action name="actionDumpRAM">
   <property name="text">
    <string>Dump RAM...</string>
   </property>
  </action>
  <action name="actionDumpVRAM">
   <property name="text">
    <string>Dump VRAM...</string>
   </property>
  </action>
  <action name="actionDumpSPURAM">
   <property name="text">
    <string>Dump SPU RAM...</string>
   </property>
  </action>
  <action name="actionDebugShowGPUState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show GPU State</string>
   </property>
  </action>
  <action name="actionDebugShowCDROMState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show CD-ROM State</string>
   </property>
  </action>
  <action name="actionDebugShowSPUState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show SPU State</string>
   </property>
  </action>
  <action name="actionDebugShowTimersState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Timers State</string>
   </property>
  </action>
  <action name="actionDebugShowMDECState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show MDEC State</string>
   </property>
  </action>
  <action name="actionDebugShowDMAState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show DMA State</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset theme="screenshot-2-line"/>
   </property>
   <property name="text">
    <string>&amp;Screenshot</string>
   </property>
  </action>
  <action name="actionMemoryCardSettings">
   <property name="icon">
    <iconset theme="memcard-line"/>
   </property>
   <property name="text">
    <string>&amp;Memory Cards</string>
   </property>
  </action>
  <action name="actionResumeLastState">
   <property name="icon">
    <iconset theme="play-circle-line"/>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
   <property name="toolTip">
    <string>Resumes the last save state created.</string>
   </property>
  </action>
  <action name="actionViewToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Toolbar</string>
   </property>
  </action>
  <action name="actionViewLockToolbar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lock Toolbar</string>
   </property>
  </action>
  <action name="actionViewStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Status Bar</string>
   </property>
  </action>
  <action name="actionViewGameList">
   <property name="icon">
    <iconset theme="list-check"/>
   </property>
   <property name="text">
    <string>Game &amp;List</string>
   </property>
  </action>
  <action name="actionViewSystemDisplay">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>System &amp;Display</string>
   </property>
  </action>
  <action name="actionViewGameProperties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="file-settings-line"/>
   </property>
   <property name="text">
    <string>Game &amp;Properties</string>
   </property>
  </action>
  <action name="actionMemoryCardEditor">
   <property name="text">
    <string>Memory &amp;Card Editor</string>
   </property>
  </action>
  <action name="actionCPUDebugger">
   <property name="text">
    <string>CPU D&amp;ebugger</string>
   </property>
  </action>
  <action name="actionViewGameGrid">
   <property name="icon">
    <iconset theme="function-line"/>
   </property>
   <property name="text">
    <string>Game &amp;Grid</string>
   </property>
  </action>
  <action name="actionMergeDiscSets">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Merge Multi-Disc Games</string>
   </property>
  </action>
  <action name="actionGridViewShowTitles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Titles (Grid View)</string>
   </property>
  </action>
  <action name="actionGridViewZoomIn">
   <property name="text">
    <string>Zoom &amp;In (Grid View)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionGridViewZoomOut">
   <property name="text">
    <string>Zoom &amp;Out (Grid View)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionGridViewRefreshCovers">
   <property name="text">
    <string>Refresh &amp;Covers (Grid View)</string>
   </property>
  </action>
  <action name="actionOpen_Memory_Card_Directory">
   <property name="text">
    <string>Open Memory Card Directory...</string>
   </property>
  </action>
  <action name="actionOpenDataDirectory">
   <property name="text">
    <string>Open Data Directory...</string>
   </property>
  </action>
  <action name="actionPowerOffWithoutSaving">
   <property name="icon">
    <iconset theme="close-line"/>
   </property>
   <property name="text">
    <string>Power Off &amp;Without Saving</string>
   </property>
  </action>
  <action name="actionStartFullscreenUI">
   <property name="icon">
    <iconset theme="tv-2-line"/>
   </property>
   <property name="text">
    <string>Start Big Picture Mode</string>
   </property>
  </action>
  <action name="actionStartFullscreenUI2">
   <property name="icon">
    <iconset theme="tv-2-line"/>
   </property>
   <property name="text">
    <string>Big Picture</string>
   </property>
  </action>
  <action name="actionCoverDownloader">
   <property name="text">
    <string>Cover Downloader</string>
   </property>
  </action>
  <action name="actionMemoryScanner">
   <property name="text">
    <string>Memory &amp;Scanner</string>
   </property>
  </action>
  <action name="actionShowGameIcons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Game Icons (List View)</string>
   </property>
  </action>
  <action name="actionMediaCapture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Media Ca&amp;pture</string>
   </property>
  </action>
  <action name="actionOpenTextureDirectory">
   <property name="text">
    <string>Open Texture Directory...</string>
   </property>
  </action>
  <action name="actionReloadTextureReplacements">
   <property name="text">
    <string>Reload Texture Replacements</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/duckstation-qt.qrc"/>
 </resources>
 <connections/>
</ui>
